### ISSUE TITLE
Unexpected identifier error

### ISSUE THREAD

**Issue by pedrowindisch** [getspooky/create-express-app#1]
Whenever I try to start my server, I get an "Unexpected identifier" error. A screenshot is attached.

![image](https://user-images.githubusercontent.com/30203228/80049547-8f9e8b00-84e9-11ea-89fd-e54a1d47ce46.png)

When using WSL, however, a different error occurs.

![image](https://user-images.githubusercontent.com/30203228/80049565-a04f0100-84e9-11ea-8fca-a643fbeb398b.png)

I'm using the ES6 template.

**Comment by pedrowindisch** [comment]
Ok, I got it. The template supports ES6 modules, but the express-artisan package doesn't.

To fix that, my solution was to add the [esm](https://www.npmjs.com/package/esm) package and use it on the express-artisan module, like this:

```js
// node_modules/express-artisan/index.js
[...]

require = require('esm')(module);

require('./expressArtisan');
``` 

The ESM package will allow every Node.js version to support ES6 modules, including those who don't support them natively (<13.12.0).

Everything's working perfectly now. I tested it with the other templates and they're working fine. If you want, I can make a PR with my fix, but I don't know if this is a general issue or if this is happening only on my PC.
